tests/ccl_test_correlation_3d.c

```diff
@@ -83,9 +83,9 @@ static void compare_correlation_3d(int i_model,struct corrs_3d_data * data)
   nr=linecount(f)-1; rewind(f);
 
   // tolerence on abs difference in r^2 xi(r) for the range r = 0.1 - 100 Mpc (40 points in r) for z=0,1,2,3,4,5
-  double CORR_TOLERANCE1[6] = {2.84e-2, 3.57e-3, 1.37e-3, 1.26e-3, 2.0e-4, 1.8e-4};
+  double CORR_TOLERANCE1[6] = {4.0e-2, 1.0e-2, 5.0e-3, 4.0e-3, 1.0e-3, 8.0e-4};
   // tolerence on abs difference in r^2 xi(r) for the range r = 50 - 250 Mpc (100 points in r) for z=0,1,2,3,4,5
-  double CORR_TOLERANCE2[6] = {3.15e-2, 9.87e-3, 5.04e-3, 3.05e-3, 9.98e-4, 7.14e-4};
+  double CORR_TOLERANCE2[6] = {4.0e-2, 1.0e-2, 5.0e-3, 4.0e-3, 1.0e-3, 8.0e-4};
```

Comment:
Should we be concerned about these tolerances going up?

Reply:
These were set originally to **exactly** the tolerances that were achieved at the time. When the P(k) splines change, they move up and down slightly, so I've just made them all larger to allow future changes in the P(k) part from affecting this test. 
We're trying to test the P(k) -> xi(r) code here, so this test should be robust against changes in the P(k) calculation.

Reply:
Right. This test is pretty useless TBH.